promgen/models.py

```diff
@@ -473,7 +473,7 @@ def copy_to(self, content_type, object_id):
             self.name = '{}_{}'.format(self.name, slugify(content_object.name)).replace('-', '_')
             self.content_type = content_type
             self.object_id = object_id
-            self.enabled = False
+            self.enabled = True
```

Comment:
The original choice of `False` was probably arbitrarily chosen, but let us go ahead and add a comment either here, or as part of the function header to explain our default of True.

Reply:
Done in 0c18e28.